### DISCUSSION TITLE
Placeholder image

### DISCUSSION THREAD

**Discussion by fsask** [pods-framework/pods#6712] [Ideas]
## Expected Behavior

 

It would be useful to be able to have a "placeholder" image in the field type of Image. That way, instead of having a blank area (no image) a placeholder image would appear.

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## WordPress Environment

<details>
```
Copy and paste your System Details from **Pods Admin > Settings > Debug Information** in WordPress admin here.
```
</details>

## Pods Package Export (helpful!)

```
Copy and Paste the JSON Export from **Pods Admin, Migrate: Packages, Export** in WordPress admin here
```

[a comment by an automated account is not shown]

**Comment by jimtrue** [comment]
@fsask This honestly sounds like something you should handle at the theme level. You're using Pods for your content management. Display strategy, ie, placeholder images is part of your content strategy and that's something you would handle as an output fallback if the image didn't exist. 

That's just my viewpoint. I'm open to reasons why this should be handled at a Pods level.

**Comment by fsask** [comment]
It's really just a question of user input. I have a client who has little technical skill, but they are able to go through a Pods form and fill in the fields. I have set the Pod up so that they must choose an option for certain fields, on others (not required fields) [if] statements just hide the empty fields - no problem. But the images are slightly different, I guess. 

By having an automatically applied placeholder image, the space the image takes up is not an issue on the page display, as as long as there is an image of any kind (they're all set to the same size) the formatting is preserved. If the field containds nothing, it becomes more difficult to preserve page formatitng, especially over different viewing platform/device screen sizes.

**Comment by sc0ttkclark** [comment]
Have you tried entering the attachment ID into the default value field for the field?

**Comment by jimtrue** [comment]
Meaning the attachment ID of your placeholder image. That would give you the same result, I think.

**Comment by jimtrue** [comment]
Also if there's no image, you could do an if block for the image. If none was uploaded, you could output your default placeholder image in the else portion of the if block.

**Comment by JoryHogeveen** [comment]
Added need votes for the UI part (default field value as an image picker).
